### PR TITLE
rewrite README.md

### PR DESCRIPTION
**Pull request by landru29**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,55 @@
...
+ 
+# Installation
 
-```css
-a.disabled {
-    background-color: grey;
-    cursor: not-allowed;
-}
+bower install ssh://git@github.com:ovh-ux/angular-a-disabled.git --save
+
```
Comment (ghost) [inline README.md]: :warning: don't forget npm ;)

File: README.md [hunk 2]
```diff
@@ -1,31 +1,55 @@
-# angular-a-disabled
```
Comment (ghost) [inline README.md:1]: may-be we can add :skull: or :skull_and_crossbones: or :warning: to say it's deprecated

File: README.md [hunk 3]
```diff
@@ -1,31 +1,55 @@
...
+ 
+```bash
+    git clone https://github.com/ovh/angular-a-disabled.git
+    cd angular-a-disabled
+    npm install
+    bower install
+```
+ 
+You've developed a new cool feature ? Fixed an annoying bug ? We'd be happy
+to hear from you !
```
Comment (ghost) [inline README.md:40]: Oh we can refere to contrib.md

**Review by ghost — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (KamiAmiga) [reply README.md:1]: I'd say ⚠️ is more understandable

**Review by ghost — APPROVED** [review]
Ok for me

**Pull request merged**